### PR TITLE
Bumped version numbers

### PR DESCRIPTION
**Pull request by Cali0707**

### CODE REVIEW

**Review by kcussen — APPROVED** [review]
Not sure this'll work but decently sure it won't break anything!

**Comment by kcussen** [comment]
@fsayyid can you merge this after 2nd review? I've added you as as the merge-master on main.

**Review by fsayyid — APPROVED** [review]

**Pull request merged**
